### DISCUSSION TITLE
How do I disable players from claiming in my spawn??/

### DISCUSSION THREAD

**Discussion by zachcronin63** [GriefPrevention/GriefPrevention#869] [Q&A] [answered]
This question is simple, how do i disable players from making a claim in my spawn?? I use worldguard to protect my spawn? if anyone could please help, let me know thank you so much!

**Comment by bigpresh** [comment, marked as answer]
Easiest way is just make an admin claim over the whole area you want to reserve.

**Comment by RoboMWM** [comment]
^
